dask/dataframe/tests/test_dataframe.py

```diff
@@ -1992,14 +1992,42 @@ def test_repartition_freq_errors():
 
 
 def test_repartition_freq_month():
-    ts = pd.date_range("2015-01-01 00:00", " 2015-05-01 23:50", freq="10min")
+    ts = pd.date_range("2015-01-01 00:00", "2015-05-01 23:50", freq="10min")
     df = pd.DataFrame(
         np.random.randint(0, 100, size=(len(ts), 4)), columns=list("ABCD"), index=ts
     )
-    ddf = dd.from_pandas(df, npartitions=1).repartition(freq="1M")
+    ddf = dd.from_pandas(df, npartitions=1).repartition(freq="MS")
```

Comment:
Any reason you changed this? Should we also have a test for month-end?

Reply:
I think there is no use case for `freq="1M"`, since the divisions (except for the last one) indicate the **minimum** and should therefore correspond with the **start** of a period.

Say you do `freq="1M"`, then e.g. `pd.Timestamp("2020-1-31 00:00:01")` will end up in a partition with e.g. `pd.Timestamp("2020-2-1 12:00:00")`. 

So again: I cannot think of a use case where you like to make dask partitions based on month-end, but that could be my own lack of imagination.